CHANGES.rst

```diff
@@ -6,6 +6,12 @@ There's a frood who really knows where his towel is.
 2.3 (unreleased)
 ^^^^^^^^^^^^^^^^
 
+- Flake8 cleanup
```

Comment:
@hvelarde no opposition about this, is not really useful.
It take me one minute for remove the entry